doc/en/user/source/data/cascaded/wms.rst

```diff
@@ -62,6 +62,18 @@ Connecting a remote WMS allows for the following features:
 
 * Full **REST Configuration**. See the :ref:`rest` section for more information about the GeoServer REST interface.
 
+Cascaded WMS Settings
+---------------------
+
+Making use of Remotely advertised styles and supported image formats.
+
+* **Remote Styles Configuration**. Remote Styles advertised in WMS capability document under <Layer> tag, can also be used. A default style and additionally supported styles can be selected. By default no remote style is selected which indicates Geoserver to use whatever style is configured remotely and all available styles are selected. This means that remote styles can be passed in a GetMap request just like local styles. If the styles on remote WMS server have changed, please re-save the layer from UI.
+
+* **Remote Image Format**. Preferred image format(s) can be selected. It is possible to select a preferred image format and additionally supported image formats. This configuration works looks at the requested image format in local GetMap, if the GetMap format is either the preferred remote format or one of the many selected remote formats, the passed image format will be relayed in the remote WMS request. If the image format requested in local GetMap is neither the preferred remote image format nor in the list of Selected formats, the remote WMS format will use the preferred remote image format. This setting only works for image formats and ignore other advertised formats such as JSON, KML and SVG etc
```

Comment:
Does this mean that if the local GetMap uses JPEG the remote request will do as well? Hmm... this would cause double JPEG compression and thus quality loss.

Reply:
yes that's the case, if remote supports PNG then simply remove JPEG from the Selected list